views/layout.ejs

```diff
@@ -45,7 +45,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <!--STYLES-->
-    <link rel="stylesheet" href="/styles/importer.css">
+    <link rel="stylesheet" href="/styles/importer.1655727270763.css">
```

Comment:
I'm curious about how this works. Is the datetime integer `1655727270763` arbitrary?

Reply:
Yeah it's a bit weird and I don't entirely know how it works but I have an idea.

1 - All the css/scss in the public folder is moved to a .tmp folder https://github.com/UKForeignOffice/loi-application-service/blob/develop/tasks/pipeline.js#L56
This is because on every build everything here is wiped and rebuilt.

2 - The code looks for some html comments, in this case `<!--STYLES-->` and `'<!--STYLES END-->'` and injects all the css files into that part of the html file.
https://github.com/UKForeignOffice/loi-application-service/blob/develop/tasks/config/sails-linker.js#L84

But it's a super old way of doing things. I don't think I've seen anything like this before.